Microsoft/AppCenter-SDK-Apple
# Add UI to test overriding common schema logs and fix UI glitches

* add UI to test common schema logs
* make sure keyboard for editing properties can be dismissed by dragging the table view or hitting return on the keyboard
* make sure the last cell in the table view is no cur off by a few pixels
